data/Kocacinar_2003/metadata.yml

```diff
@@ -0,0 +1,248 @@
+source:
+  primary:
+    key: Kocacinar_2003
+    bibtype: Article
+    year: '2003'
+    author: F. Kocacinar and R. F. Sage
+    journal: Plant, Cell & Environment
+    title: Photosynthetic pathway alters xylem structure and hydraulic function in
+      herbaceous plants
+    volume: '26'
+    number: '12'
+    pages: 2015--2026
+    doi: 10.1111/j.1365-2478.2003.01119.x
+contributors:
+  data_collectors:
+  - last_name: Kocacinar
+    given_name: Ferit
+    ORCID: 0000-0003-0890-6117
+    affiliation: Kahramanmaras Sütcü Imam University, Turkey
+    notes: no contact
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code:         '
+    data %>% mutate(location = "University of Toronto, Ontario")
+  '
+  collection_date: unknown/2003
+  taxon_name: taxon_name
+  location_name: location
+  description: A comparison of stem hydraulic conductivity and vascular anatomy between
+    eight C3 and eight C4 herbaceous species.
+  basis_of_record: experiment
+  life_stage: saplings
+  sampling_strategy: All plants were grown from seeds for three months in 6 L pots
+    on an outdoor roof at the University of Toronto, Ontario. The potting medium was
+    a mix of 25% Promix (Sun Gro Horticulture Canada Ltd, Seba Beach, AB, Canada),
+    25% sand and 50% topsoil. The plants were watered regularly to avoid drought and
+    were fertilized once a week with a full-strength Hoagland's solution. Plants were
+    measured approximately 3 months after germination.
+  original_file: none, transcribed from paper
+  notes: Only the species with Australian seed sources are included.
+locations:
+  University of Toronto, Ontario:
+    latitude (deg): 43.67
+    longitude (deg): -79.42
+contexts:
+- context_property: seed provenance
+  category: treatment
+  var_in: locality
+  values:
+  - value: Australian National University
```

Comment:
Remove Australian National University as all of the species are from Darwin, ANU was just for one species non-native to Australia I believe

Reply:
done